### PR TITLE
Test and fix UFS mounts from fstab into a jail

### PR DESCRIPTION
**Pull request by gronke**
- Test ability to mount UFS volumes into a jail

FYI @FabianFreyer

### CODE REVIEW

File: tests/test_Fstab.py [hunk 1]
```diff
@@ -87,4 +93,33 @@ def test_parses_auto_lines(
...
+		name = f"{root_dataset.name}/zvol{r}"
+
+		root_dataset.pool.create(
+			dataset_name,
+			fsopts=dict(volsize="16M"),
+			fstype=libzfs.DatasetType.VOLUME
+		)
+
+		dataset = zfs.get_dataset(dataset_name)
+		yield dataset
```
Comment (fabianfreyer) [inline tests/test_Fstab.py]: should we not wrap this generator with something that calls `newfs`?

Comment (gronke) [reply tests/test_Fstab.py]: Would you agree that another fixture `zfs_volume_ufs` could do that job?

Comment (fabianfreyer) [reply tests/test_Fstab.py]: probably a good idea

Comment (gronke) [reply tests/test_Fstab.py]: done

**Pull request merged**